tests/models/models_for_test_gas_stock_2_elec_compressor.yml

```diff
@@ -0,0 +1,140 @@
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+library:
+  id: basic
+  description: Basic library
+
+  port-types:
+    - id: flow
+      description: A port which transfers power flow
+      fields:
+        - name: flow
+  models:
+    - id: repartition_key
+      description: A repartition key
+      parameters:
+        - name: alpha
+          time-dependent: false
+          scenario-dependent: false
+      ports:
+        - name: flow_k
+          type: flow
+      binding-constraints:
+        - name: sum
+          expression: sum_connections(flow_k.flow) = 0
```

Comment:
I think the syntax for `sum(alpha)` is not defined in lower level for now, we need to add parameters in the sum operator